### ISSUE TITLE
Firehose requests 100k index when only configured for 10k

### ISSUE THREAD

**Issue by matthewdarwin** [streamingfast/firehose-ethereum#87]
The firehose endpoint has

```
common-index-block-sizes: 10000
```

When I look at the s3 logs, I see requests for 100000 index sizes

```
GET /eth-sf-v4-block-index/0016800000.100000.combined.idx HTTP/1.1" 404 
```

Firehose should not request indexes that it is not configured to use.

**Comment by sduchesneau** [comment]
Fixed in firehose-core, `commit 75636557dda29dd4a55ccc413b525d9f2d57d91b` 
the issue was with parsing this value from the yaml file, it worked on command-line flag...

**Comment by sduchesneau** [comment]
4141bef
